CONTRIBUTING.md

```diff
@@ -70,33 +67,56 @@ Each function must have a corresponding test. If the function has parameters tha
 
 The function must contain a docstring explaining what the function does and a description of each argument. [See this example.](https://github.com/ploomber/sklearn-evaluation/blob/8056bc31ec5e372102d0ee5ada988e380b077c4b/src/sklearn_evaluation/plot/classification.py#L143)
 
-Furthermore, a full example (under the docstring's `Examples` section)must be included in the examples section of the docstring. Such an example must be standalone so that copy-paste should work. [See this example.](https://sklearn-evaluation.readthedocs.io/en/latest/api/plot.html#sklearn_evaluation.plot.confusion_matrix) Note that these examples are automatically tested by the CI.
+Furthermore, a full example must be included in the examples section of the docstring. Such an example must be standalone so that copy-paste should work. [See this example.](https://sklearn-evaluation.readthedocs.io/en/latest/api/plot.html#sklearn_evaluation.plot.confusion_matrix)
 
-Here's a docstring template you can use:
+## Telemetry : Monitoring the state of `sklearn-evaluation`
+
+`SKLearnEvaluationLogger` decorator wraps `telemetry log_api` functionality and allows to generate logs as follows:
 
 ```python
-def my_plotting_function(y_true, y_pred, ax=None):
-    """Plot {plot name}
+@SKLearnEvaluationLogger.log(feature='plot')
+def confusion_matrix(
+        y_true,
+        y_pred,
+        target_names=None,
+        normalize=False,
+        cmap=None,
+        ax=None,
+        **kwargs):
+pass
+```
+
+this will generate the following log:
+
+```json
+        {
+          "metadata": {
+          "action": "confusion_matrix"
+          "feature": "plot",
+          "args": {
+                        "target_names": "None",
+                        "normalize": "False",
+                        "cmap": "None",
+                        "ax": "None"
+                    }
+          }
+        }
+```
 
-    Parameters
```

Comment:
looks like this whole section was removed. did you rebase?

Reply:
Fixed it. My forked repo wasn't synced with the latest version.